docs/src/main/sphinx/release/release-372.md

```diff
@@ -0,0 +1,140 @@
+# Release 372 (30 Feb 2022)
+
+## General
+
+* Add :func:`trim_array` function. ({issue}`11238`)
+* Improve handling of prepared statements with long query text by compressing
+  them within HTTP headers. This can be configured or disabled using the
+  ``protocol.v1.prepared-statement-compression.length-threshold`` configuration
+  property. ({issue}`11098`)
+* Improve performance of specific queries which compare table columns of type
+  `timestamp` with `date` literals. ({issue}`11170`)
+* Add redirection awareness for `ADD COLUMN`, `DROP TABLE`, `COMMENT` tasks.
+  ({issue} `11072`)
+* Remove support for reserved memory pool. Configuration property
+  `experimental.reserved-pool-disabled` can no longer be used. ({issue}`6677`)
+* Ensure memory is released completely after query completion. ({issue}`11030`)
+* Fix certain queries failing due to dictionary compacting error.
+  ({issue}`11080`)
+* Fix `SET SESSION` and `RESET SESSION` not working for catalogs which include
+  special characters in their name. ({issue}`11171`)
+
+## Security
+
+* Allow configuration of connetct and read timeouts for LDAP authentication.
+  ({issue}`10925`)
+
+## Docker image
+
+* Add a healthcheck to the Docker container image. ({issue}`10413`)
+
+## JDBC driver
+
+* Fix `DatabaseMetaData#getTables` and `DatabaseMetaData#getColumns` to include
+  views for Iceberg, Raptor, Accumulo and Blackhole connectors.
+  ({issue}`11063`, {issue}`11060`)
+
+## Base-JDBC connector library
+
+* Fix spurious query failures when metadata cache is not enabled, and data
+  access depends on the session state. ({issue}`11068`)
+
+## Accumulo connector
+
+* Fix incorrect date results when querying `date` type columns. ({issue}`11055`)
+
+## Cassandra connector
+
+* Fix incorrect results when filtering partition keys without projections.
+  ({issue}`11001`)
+
+## ClickHouse connector
+
+* Fix spurious query failures when metadata cache is not enabled, and extra
+  credentials with `user-credential-name` or `password-credential-name` are used
+  to access data. ({issue}`11068`)
+
+## Druid connector
+
+* Fix spurious query failures when metadata cache is not enabled, and extra
+  credentials with `user-credential-name` or `password-credential-name` are used
+  to access data. ({issue}`11068`)
+
+## Hive connector
+
+* Allow redirects of Iceberg or Delta tables which have no ``StorageDescriptor``
+  in the Glue metastore. ({issue}`11092`)
+* Stop logging spurious failures when [storage caching](/connector/hive-caching)
+  is used. ({issue}`11101`)
+* Allow reading Zstandard compressed Avro files. ({issue}`11090`)
+* Fix incorrect query results after writes to a table with caching directory
```

Comment:
What's "caching directory lister"? That seems like an implementation detail. I would just say "when directory caching is enabled"